alerts/etcMemberDown.md

```diff
@@ -0,0 +1,39 @@
+# etcdMembersDown
+
+## Meaning
+
+This alert is fired when one or more etcd member goes down. The alert evaluates the number of etcd members that are currently down. Often, this alert was observed as part of a cluster upgrade when a master node is being upgraded and requires a reboot. 
+
+
+## Impact
+In etcd a majority of (n/2)+1 has to agree on membership changes or key-value upgrade proposals. This totally avoids any split-brain inconsistency. In the case that only one member is down in a 3-member cluster, it can still make forward progress. Then the quorum is 2 and 2 members are still alive. However, when more members are down, the cluster becomes unrecoverable.
+
+
+## Diagnosis
+
+Login to the cluster. Check health of master nodes if any of them is in `NotReady` state or not
+```bash
+$ oc get nodes -l node-role.kubernetes.io/master=
+```
+**TODO resolve private link**:
+Check if an upgrade is in progress. [upgrade is in progress](https://github.com/openshift/ops-sop/blob/master/v4/howto/managed-upgrade.md#how-to-monitor-upgrade-progress).
+```bash
+$ oc get upgrade -A
+```
```

Comment:
This part can be removed as well. It's only specific to OSD clusters and not OCP. Runbooks to be generic. For now, a simple check using `oc adm upgrade` to check upgrade status might suffice. We can add more details/specifics later on too based on feedback.